src/targeting/session.ts

```diff
@@ -0,0 +1,173 @@
+import type { Participations } from '@guardian/ab-core';
+import type { CountryCode } from '@guardian/libs';
+import { isString } from '@guardian/libs';
+import type { False, True } from '../types';
+
+/* -- Types -- */
+
+const referrers = ['facebook', 'twitter', 'reddit', 'google'] as const;
```

Comment:
Do we want to add `googleplus` here and in referrerTypes? It does seem to be one of the possible targeting values in GAM.

Reply:
It might be neater to do something [like this](https://www.typescriptlang.org/play?#code/MYewdgzgLgBATgUwGYLouAVAngBwRAYXGhgF4YBtAKBlpgG8a7mBLAEwC4YByJAQ2AIARiBABrbgBomzWgFs+UYAAsuvAcNFiAdKDlSZMAL7TmjWXXZqA5qOsAbBAYvzFKtQHcv22yAdPTOhNDcxcrHigPFigoVGcXBSVVHgB6KF0QFPjaYLNDVk4eRDY2aOzZRPcihBLojP1AnOkAXRg+CBhQSCgqKFwEeGQUdGw8Mhg+vBAkQeGR-sJiHsmBxDnUcZXp2dR5vEXuijAAVzkhVGaKACJ2K+agA) than to have these 4-5 string defined here and in  `referrerTypes` as well?
(Yes I'm plagiarising your comment from https://github.com/guardian/frontend/pull/24370 😄 )

Reply:
According to Wikipedia: [Google+ is defunct](https://en.wikipedia.org/wiki/Google%2B)

> ### Current status:
> **Defunct**: Discontinued for personal and brand accounts (April 2, 2019; 2 years ago)All users transitioned to Google Currents (G-Suite enterprise accounts)

Reply:
I didn't realize! In that case can we remove it from GAM and the comment on line 91?